test/FunctionalTests/Microsoft.OData.Edm.Tests/Microsoft.OData.Edm.Tests.csproj

```diff
@@ -38,26 +38,22 @@
     <ProjectReference Include="..\..\..\src\Microsoft.OData.Edm\Microsoft.OData.Edm.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1' or '$(TargetFramework)' == 'netcoreapp3.1'">
-     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.0" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp1.1'">
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.2" />
+  <ItemGroup>
+     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.8.0" />
   </ItemGroup>
    
   <ItemGroup>
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
   </ItemGroup>
  
-   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+   <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">

```

Comment:
Confirm whether we need this condition in the case of net8.0. It's even possible STJ dependency comes with the framework reference

Reply:
Removed.